api/src/main/java/org/eclipse/microprofile/config/spi/Converter.java

```diff
@@ -67,7 +68,7 @@
     /**
      * Configure the string value to a specified type
      * @param value the string representation of a property value
-     * @return the converted value
+     * @return the converted value or null if value is null
```

Comment:
@Emily-Jiang How would the configuration then treat exceptions from a converter? I think that it should be explicitly forbidden to throw an exception and rather return null or another value if makes sense.